models/mixpanel__sessions.sql

```diff
@@ -67,7 +69,7 @@ new_sessions as (
     
     select 
         *,
-        -- had the previous session timed out?
+        -- had the previous session timed out? either via inactivity or a new calendar day occurring
```

Comment:
Just a caps change request.
```suggestion
        -- had the previous session timed out? Either via inactivity or a new calendar day occurring
```